BuildScript/BuildTargets/Testing.targets

```diff
@@ -242,6 +242,16 @@ $(_dockerImageName) $(_nunitCallArgumentString)
       <Output TaskParameter="Value" PropertyName="_nunitResult" />
     </MSBuild.ExtensionPack.Xml.XmlFile>
 
+    <!--  NUnit2 tests display the assembly path in both the name and fullname attribute  -->
```

Comment:
Ah yes, nice workaround! I was thinking we'd have to extract the value, transform it, and then write it back. But given we already have the value we are looking for available locally, we can simplify this this :)